sample/CMakeLists.txt

```diff
@@ -1,15 +1,11 @@
 add_executable(sample sample.c common.c)
-
 target_link_libraries(sample PUBLIC tlsuv)
-target_include_directories(sample PUBLIC ${CMAKE_SOURCE_DIR}/include)
 
 add_executable(engine_test engine_test.c)
 target_link_libraries(engine_test PUBLIC tlsuv)
-target_include_directories(engine_test PUBLIC ${CMAKE_SOURCE_DIR}/include)
 
 add_executable(sample-cf sample-cf.c)
 target_link_libraries(sample-cf PUBLIC tlsuv)
-target_include_directories(sample-cf PUBLIC ${CMAKE_SOURCE_DIR}/include)
```

Comment:
all these removes are on purpose i expect, but calling out to make sure

Reply:
yes